src/core/edm/reducers/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow
+ */
+
+import { List, Map, fromJS } from 'immutable';
+import { EntitySetsApiActions } from 'lattice-sagas';
+
+import getEntityDataModelTypesReducer from './getEntityDataModelTypesReducer';
+import getEntitySetReducer from './getEntitySetReducer';
+import getEntitySetsReducer from './getEntitySetsReducer';
+import getOrSelectEntitySetsReducer from './getOrSelectEntitySetsReducer';
+
+import {
+  ENTITY_SETS,
+  ENTITY_SETS_INDEX_MAP,
+  ENTITY_TYPES,
+  ENTITY_TYPES_INDEX_MAP,
+  PROPERTY_TYPES,
+  PROPERTY_TYPES_INDEX_MAP,
+  RS_INITIAL_STATE,
+} from '../../redux/constants';
+import {
+  GET_EDM_TYPES,
+  GET_OR_SELECT_ENTITY_SETS,
+  getEntityDataModelTypes,
+  getOrSelectEntitySets,
+} from '../actions';
+
+const {
+  GET_ENTITY_SET,
+  GET_ENTITY_SETS,
+  getEntitySet,
+  getEntitySets,
+} = EntitySetsApiActions;
+
+const INITIAL_STATE :Map = fromJS({
```

Comment:
An interesting pattern. You're basically composing reducers without using `compose`. But this pattern also separates the initial state of each reducer from the actual reducer logic.